### ISSUE TITLE
click on column drag icon throws error

### ISSUE THREAD

**Issue by acthp** [acthp/ucsc-xena-client#23]
A click w/o drag causes an error due to uninitialized variable newPos.

**Comment by achang28** [comment]
Is this what you are referring to?
![cohortselection](https://cloud.githubusercontent.com/assets/3760499/13272203/48538b86-da4f-11e5-8e42-06ca2dada0ca.png)

**Comment by acthp** [comment]
Yes, as noted root cause is newPos is uninitialized if the mouse doesn't
move between mousedown & mouseup. You should see the error on the console
if you click on that button.

I believe the initial value should be the value in the last 'else'
statement in Sortable.componentWillMount. So, fix is probably to move that
value to the declaration of newPos, and drop the last 'else' statement.

b.c.

**Comment by achang28** [comment]
Here's the change I made.  Turns out that .map on line 69 does not even get invoked at all:  `Rx.DOM.fromEvent(window, 'mousemove').map(function (mm) {`.  Rightfully so, I guess, given the lack of "mousemove" events.  Thought of another way to properly handle this situation, but this was the simplest thusfar.

![sortablechange](https://cloud.githubusercontent.com/assets/3760499/13273462/97077108-da58-11e5-8ddb-85263e4060ff.jpg)
